chance/p5js/Popup-Circles/index.md

```diff
@@ -3,7 +3,7 @@ layout: p5js
 author: Daniele Maselli
 author-url: https://github.com/DanieleMaselli 
 title: Popup Circles 
-tags: [P5.js, chance, repeat, random, Law of Similarity]
+tags: [P5.js, chance, random, form, Law of Connected Elements]
```

Comment:
dear @DanieleMaselli i think we are misunderstanding us:  

```yaml
tags: 
  - P5.js
  - color
  - repeat
  - Law of Something
```

I'm totally okay with the repeat tag in there. The Laws you sort this in is a bit wired. Why connected elements? https://github.com/FH-Potsdam/laws-of-gestalt/blob/master/Law-of-Connected-Elements.md